TenthSeminarTask/create_polinom.py

```diff
@@ -0,0 +1,45 @@
+
+import os
+
+
+def first_polynom_input(nums=[]):
```

Comment:
Значение по умолчанию не может быть изменяемым. Лучше сделать так:
```suggestion
def first_polynom_input(nums=None):
    if nums is None:
        nums = []
```